examples/cpp/esp32-testbed/3_idf_flash.sh

```
@@ -0,0 +1 @@
+idf.py -p /dev/ttyACM1 flash monitor
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)